tests/utils/test_operator_utils.py

```diff
@@ -0,0 +1,62 @@
+import unittest
+
+from fivetran_provider_async.utils.operator_utils import (
+    _get_fields,
+    _get_openlineage_name,
+    _get_openlineage_namespace,
+    _get_table_id,
+    get_dataset,
+)
+from tests.common.static import (
+    MOCK_FIVETRAN_DESTINATIONS_RESPONSE_PAYLOAD,
+    MOCK_FIVETRAN_METADATA_COLUMNS_RESPONSE_PAYLOAD,
+    MOCK_FIVETRAN_METADATA_TABLES_RESPONSE_PAYLOAD,
+    MOCK_FIVETRAN_RESPONSE_PAYLOAD,
+    MOCK_FIVETRAN_SCHEMA_RESPONSE_PAYLOAD,
+)
+
+
+class TestFivetranOperator(unittest.TestCase):
```

Comment:
```suggestion
class TestFivetranUtils(unittest.TestCase):
```
same class name `TestFivetranOperator`  is used here and `operators/test_fivetran.py`